### DISCUSSION TITLE
phaser-on-nodejs with Colyseus - ENOENT kills server

### DISCUSSION THREAD

**Discussion by hatala91** [geckosio/geckos.io#128] [Help / Questions] [answered]
Hello together 👋 

Did anybody successfully use `@geckos.io/phaser-on-nodejs` with Colyseus yet? I was able to get it running 🚀 but whenever a client disconnects it kills the whole game server with this error:

```
Error: ENOENT: no such file or directory, open '.../node_modules/@geckos.io/phaser-on-nodejs/lib/node:internal/stream_base_commons'
```

I successfully used `@geckos.io/phaser-on-nodejs` for a single lobby type of game. When trying to enable multiple rooms I looked at multithreading but ran into issues with `node-canvas` so that is how I ended up here 😄 

**Update**: Nevermind, it turns out this issue only occurs when running `ts-node-dev` for serving 🤦 On the plus side, there now is a template for using `colyseus` with `phaser-on-nodejs`: https://github.com/hatala91/lizard-arena

**Comment by yandeu** [comment, marked as answer]
_Solution:_

> Update: Nevermind, it turns out this issue only occurs when running ts-node-dev for serving 🤦 On the plus side, there now is a template for using colyseus with phaser-on-nodejs: https://github.com/hatala91/lizard-arena
